### PR TITLE
Adding Integrations User Guide from AKN

### PR DESCRIPTION
**Pull request by drewavisautodesk**
Trying again... adding the content from the Integrations User Guide on AKN to the developer help.

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/01eada2acc1f00bf670fb8fc721f9e3a7c376a73/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/01eada2acc1f00bf670fb8fc721f9e3a7c376a73/index.html)

File: docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md
Comment (shaynacohen) [inline docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md:13]: This link isn't working, maybe try it without the extra dashes?

File: docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md
Comment (shaynacohen) [inline docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md:22]: This link is having the same issue, maybe it's the variable or the extra dashes?

File: docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md
Comment (shaynacohen) [inline docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md:100]: Update this link to: https://help.autodesk.com/view/SGSUB/ENU/?guid=SG_Administrator_ar_data_management_ar_linking_local_files_html

**Review by shaynacohen — COMMENTED** [review]
@drewavisautodesk - I left a couple of more minor fix notes, but it looks great! I'll leave it up to you if you want to fix before merging, or fix after the merge.

File: docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md
Comment (drewavisautodesk) [reply docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md:13]: I think it's the include variable inside the link, I'll see if I can replace that with the variable

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/9c0b73352699cf903f9231cb942caf3f3724fb05/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/43a389dc1b08f0af9cb7ff3dbe8067fa3c86c665/index.html)

File: docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md
Comment (drewavisautodesk) [reply docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md]: I've had to remove the product name variable completely from the title in order to fix this issue.

File: docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md
Comment (drewavisautodesk) [reply docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md]: Some solution here.

File: docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md
Comment (drewavisautodesk) [reply docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md]: Link updated

**Review by shaynacohen — APPROVED** [review]

**Comment by shaynacohen** [comment]
LGTM!

**Pull request merged**
